commands/droplets.go

```diff
@@ -556,9 +556,35 @@ func matchDroplets(ids []string, ds do.DropletsService, fn matchDropletsFn) erro
 
 // RunDropletGet returns a droplet.
 func RunDropletGet(c *CmdConfig) error {
+
 	id, err := getDropletIDArg(c.NS, c.Args)
 	if err != nil {
-		return err
+		// Try to find Droplet by name
+		if len(c.Args) < 1 {
```

Comment:
```suggestion
		if len(c.Args) != 1 {
```

The get command only returns one result, so should only accept one argument.